input.go

```diff
@@ -174,6 +180,8 @@ func IsGamepadButtonPressed(id int, button GamepadButton) bool {
 }
 
 // TouchIDs returns the current touch states.
+// If you want to know whether the TouchIDs started being pressed in the current frame,
```

Comment:
Add a blank line above.

s/TouchIDs/a touch/

Reply:
Is it s/the TouchIDs/a touch/  ?

Reply:
Ah yes, right.

Reply:
fix on af604ff18148f605449be91ecb59a99e5bb08e71